### PR TITLE
#4 fw cloud.report export

### PR DESCRIPTION
**Pull request by KAR2022**
Overview

Script to create report export table.

Connects #4

Checklist
fw_cloud.report_export.sql
report_export.py

Notes
Check to see if fw_cloud schema exists.

### CODE REVIEW

File: report_export.py [hunk 1]
```diff
@@ -48,9 +55,10 @@
 for project in fw.projects.iter():
     pid = project.id
     name = project.label
-    print(f"Fetching view for project {name} ({pid})...")
-    d = json.load(fw.read_view_data(view, pid, decode=False, format="json-flat"))
-    all_data.extend(d)
+    if name =='CBTN_reports':
```
Comment (afamiliar) [inline report_export.py:58]: it's much more computationally efficient if the for loop is removed & hard-code the `pid` and `name` for the `CBTN_reports` project (so then wont iterate through the numerous projects on Flywheel)

File: report_export.py [hunk 2]
```diff
@@ -17,7 +17,8 @@
 
 fw = flywheel.Client(fw_api_token)
 db = create_engine(db_url)
-table = "flywheel_export"
+table = "report_export"
```
Comment (afamiliar) [inline report_export.py:20]: the table name should indicate the source as Flywheel (to differentiate it from other tables in the warehouse) - maybe something like `flywheel_report_export` and changing other tables to `flywheel_mri_export` etc.

File: fw_cloud.report_export.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    file_classification_measurement text COLLATE pg_catalog."default",
+    file_info_dim1 double precision,
+    file_info_dim2 double precision,
+    file_info_spacing_between_slices double precision,
+    file_info_manufacturer text COLLATE pg_catalog."default",
+    file_info_manufacturers_model_name text COLLATE pg_catalog."default",
+    file_info_software_versions varchar ,
+    file_created text COLLATE pg_catalog."default",
+    file_modified text COLLATE pg_catalog."default"
+)
```
Comment (afamiliar) [inline fw_cloud.report_export.sql:26]: a lot of these fields are specific to the MRIs and not necessary to include for reports. please remove:

-     file_modality text COLLATE pg_catalog."default",
-     file_info_magnetic_field_strength double precision,
-     file_classification_intent text COLLATE pg_catalog."default",
-     file_classification_features text COLLATE pg_catalog."default",
-     file_classification_measurement text COLLATE pg_catalog."default",
-     file_info_dim1 double precision,
-     file_info_dim2 double precision,
-     file_info_spacing_between_slices double precision,
-     file_info_manufacturer text COLLATE pg_catalog."default",
-     file_info_manufacturers_model_name text COLLATE pg_catalog."default",
-     file_info_software_versions varchar ,

**Review by afamiliar — CHANGES_REQUESTED** [review]
see comments

File: report_export.py (hunk 1 above)
Comment (KAR2022) [reply report_export.py]: Makes sense. 
Thanks for reviewing. Fixed the code.

File: report_export.py (hunk 2 above)
Comment (KAR2022) [reply report_export.py]: Changed the names to :
flywheel_mri_export
flywheel_report_export
flywheel_slide_export

File: fw_cloud.report_export.sql (hunk 3 above)
Comment (KAR2022) [reply fw_cloud.report_export.sql]: I agree. Made the changes to the script and also on the table.

**Pull request merged**